src/framing.c

```diff
@@ -1492,6 +1492,22 @@ const int head3_7[] = {0x4f,0x67,0x67,0x53,0,0x05,
                        1,
                        0};
 
+int compare_packet(const ogg_packet *op1, const ogg_packet *op2){
+  if (op1->packet!=op2->packet)
+    return(1);
```

Comment:
Sounds good. I've made these changes and they pass ``make check``, so I'll update the PR.